php/recipes/module-aws_elasticache_cluster_client.rb

```diff
@@ -0,0 +1,20 @@
+include_recipe 'php::dependencies-ppa'
+
+ext = 'amazon-elasticache-cluster-client.so'
+
+source_version = node['amazon-elasticache-cluster-client']['php_version']
+
+cookbook_file "/usr/lib/php/20151012/#{ext}" do
```

Comment:
@till Not sure about that one - either this lib path is "hardcoded" in the client, so we should bail out if the php ext path is different, or this should be done programmatically